src/scripts/core/contour-utils.js

```diff
@@ -379,30 +379,46 @@
 
             var niceMinMax = axisHelpers.niceMinMax(dataMin, dataMax, ticks, zeroAnchor);
 
-            return [niceMinMax.min, niceMinMax.max];
+            // return [niceMinMax.min, niceMinMax.max];
 
             // // we want null || undefined for all this comparasons
             // // that == null gives us
-            // if (min == null && max == null) {
-            //     return [niceMinMax.min, niceMinMax.max];
-            // }
+            if (min == null && max == null) {
+                return [niceMinMax.min, niceMinMax.max];
+            }
 
-            // if (min == null) {
-            //     return [Math.min(niceMinMax.min, max), max];
-            // }
+            if (min == null) {
+                return [Math.min(niceMinMax.min, max), max];
+            }
 
-            // if (max == null) {
-            //     return [min, Math.max(min, niceMinMax.max)];
-            // }
+            if (max == null) {
+                return [min, Math.max(min, niceMinMax.max)];
+            }
 
-            // return [min, max];
+            return [min, max];
         },
 
-        niceTicks: function (min, max, ticks, zeroAnchor) {
+        niceTicks: function (min, max, ticks, zeroAnchor, opts) {
             ticks = ticks == null ? 5 : ticks;
 
             var niceMinMax = axisHelpers.niceMinMax(min, max, ticks, zeroAnchor);
-            return niceMinMax.tickValues;
+            var tickValues = niceMinMax.tickValues;
```

Comment:
'min' and 'max' are the first 2 parameters, but it's also looking at `min` and `max` from options? That's too confusing; have the parent determine the right `min` `max` to operate on.